product-web-app/package.json

```diff
@@ -15,11 +15,14 @@
     "test": "ts-mocha"
   },
   "dependencies": {
+    "@govuk-prototype-kit/common-templates": "^1.1.1",
     "axios": "1.4.0",
     "cookie": "^0.5.0",
     "dotenv": "16.3.1",
     "express": "4.18.2",
     "express-session": "1.17.3",
+    "govuk-frontend": "^4.6.0",
+    "govuk-prototype-kit": "^13.13.3",
```

Comment:
Remove that change, update package-lock afterwards
```suggestion
    "axios": "1.4.0",
    "cookie": "^0.5.0",
    "dotenv": "16.3.1",
    "express": "4.18.2",
    "express-session": "1.17.3"
```

Reply:
Addressed.